### ISSUE TITLE
Unicode characters displaying little boxes with "?"

### ISSUE THREAD

**Issue by mchitre** [zellij-org/zellij#1034]
Running on MacOS and Terminal or iTerm2 ssh, I get:

<img width="387" alt="image" src="https://user-images.githubusercontent.com/3949878/151737155-0314565f-92d3-4fc8-9786-1d225356d3b3.png">

Also, when I type unicode characters in the terminal, they display twice in different colors, and the cursor placement messes up:

<img width="350" alt="image" src="https://user-images.githubusercontent.com/3949878/151737256-f76613bb-3d0e-474e-86ad-22c29e61c926.png">

(this was after having typed just one "β".

In the same terminal, without zellij, unicode works fine.

Any ideas?

**Comment by imsnif** [comment]
Did you try https://zellij.dev/documentation/compatibility.html#the-status-bar-fonts-dont-render-correctly ?

**Comment by mchitre** [comment]
Thanks for the quick pointer. Managed to fix it -- problem was missing glyphs in font used by iTerm2. Changing the font fixed it.

**Comment by mchitre** [comment]
Actually ... fixed the first issue, not the second. The strange display of "β" still persists. Reopening and changing the issue title to reflect this.

**Comment by imsnif** [comment]
> Actually ... fixed the first issue, not the second. The strange display of "β" still persists. Reopening and changing the issue title to reflect this.

Right. So could I ask that you fill in the details in our ISSUE_TEMPLATE.md? specifically the Zellij version, the attached log file and the `tput cols` and `tput lines` output from inside the pane with the bug. That'll allow us to recreate this.

**Comment by mchitre** [comment]
**Basic information**

`zellij --version`: `zellij 0.24.0`
`tput lines`: `48`
`tput cols`: `374`
`uname -av` or `ver`(Windows): `Linux arl-X570-ROG 5.11.0-40-generic #44~20.04.2-Ubuntu SMP Tue Oct 26 18:07:44 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`

List of programs you interact with:
- none to reproduce the bug, just start zellij and type `β`

**Further information**

When I type unicode characters in the terminal, they display twice in different colors, and the cursor placement messes up. See screenshot in issue description above.

Logs attached:
[zellij-logs.zip](https://github.com/zellij-org/zellij/files/7978423/zellij-logs.zip)
